src/test/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/CreditorsAfterOneYearServiceImplTests.java

```diff
@@ -0,0 +1,191 @@
+import uk.gov.companieshouse.api.handler.smallfull.SmallFullResourceHandler;
+import uk.gov.companieshouse.api.handler.smallfull.creditorsafteroneyear.CreditorsAfterOneYearResourceHandler;
+import uk.gov.companieshouse.api.handler.smallfull.creditorsafteroneyear.request.CreditorsAfterOneYearGet;
+import uk.gov.companieshouse.api.model.accounts.smallfull.creditorsafteroneyear.CreditorsAfterOneYearApi;
+import uk.gov.companieshouse.web.accounts.api.ApiClientService;
+import uk.gov.companieshouse.web.accounts.exception.ServiceException;
+import uk.gov.companieshouse.web.accounts.model.smallfull.BalanceSheet;
+import uk.gov.companieshouse.web.accounts.model.smallfull.BalanceSheetHeadings;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.creditorsafteroneyear.CreditorsAfterOneYear;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.creditorsafteroneyear.OtherCreditors;
+import uk.gov.companieshouse.web.accounts.model.smallfull.notes.creditorsafteroneyear.Total;
+import uk.gov.companieshouse.web.accounts.service.smallfull.BalanceSheetService;
+import uk.gov.companieshouse.web.accounts.service.smallfull.CreditorsAfterOneYearService;
+import uk.gov.companieshouse.web.accounts.transformer.smallfull.CreditorsAfterOneYearTransformer;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class CreditorsAfterOneYearServiceImplTests {
+
+    @InjectMocks
+    private CreditorsAfterOneYearService mockCreditorsAfterOneYearService = new CreditorsAfterOneYearServiceImpl();
+
+    @Mock
+    CreditorsAfterOneYearTransformer mockTransformer;
+
+    @Mock
+    private CreditorsAfterOneYearResourceHandler mockCreditorsAfterOneYearResourceHandler;
+
+    @Mock
+    BalanceSheetService mockBalanceSheetService;
+
+    @Mock
+    BalanceSheet mockBalanceSheet;
+
+    @Mock
+    private CreditorsAfterOneYearGet mockCreditorsAfterOneYearGet;
+
+    @Mock
+    private ApiClientService mockApiClientService;
+
+    @Mock
+    private ApiClient mockApiClient;
+
+    @Mock
+    private SmallFullResourceHandler mockSmallFullResourceHandler;
+
+    private static final String TRANSACTION_ID = "transactionId";
+
+    private static final String COMPANY_ACCOUNTS_ID = "companyAccountsId";
+
+    private static final String COMPANY_NUMBER = "companyNumber";
+
+    private static final String BASE_SMALL_FULL_URI = "/transactions/" + TRANSACTION_ID +
+            "/company-accounts/" + COMPANY_ACCOUNTS_ID +
+            "/small-full";
+
+    private static final String CREDITORS_AFTER_ONE_YEAR_URI = BASE_SMALL_FULL_URI + "/notes/creditors-after-more-than-one-year";
+
+
+    @Test
+    @DisplayName("GET - Creditors After One Year successful path")
+    void getCreditorsAfterOneYearSuccess() throws Exception {
+
+        CreditorsAfterOneYearApi creditorsAfterOneYearApi = new CreditorsAfterOneYearApi();
+        getMockCreditorsAfterOneYearApi(creditorsAfterOneYearApi);
```

Comment:
We could rename this to just `mockCreditorsAfterOneYearApi(creditorsAfterOneYearApi);` as we are not getting anything ..just mockint.